torch/utils/data/dataloader.py

```diff
@@ -72,7 +73,7 @@ def default_collate(batch):
         return torch.LongTensor(batch)
     elif isinstance(batch[0], float):
         return torch.DoubleTensor(batch)
-    elif isinstance(batch[0], str):
```

Comment:
Helpful "documentation" for string types in Python 2 vs 3:
https://github.com/benjaminp/six/blob/1.10.0/six.py#L40-L53

If you want `string_classes=(str,bytes)` for Py3, then you might want `string_classes=(basestring,str)` for Py2.